MekHQ/src/mekhq/campaign/finances/Loan.java

```diff
@@ -222,43 +224,44 @@ public void paidLoan() {
-     * Gray Monday event is active, a special predatory loan with significantly higher interest
-     * rates and penalties is offered. Otherwise, the loan terms are progressively better
-     * as the player's rating improves.</p>
+     * performance rating and whether the game is simulating the Gray Monday event. If the Gray Monday event is active,
+     * a special predatory loan with significantly higher interest rates and penalties is offered. Otherwise, the loan
+     * terms are progressively better as the player's rating improves.</p>
      *
      * <p>The returned {@link Loan} object contains all relevant terms such as the principal,
      * interest rate, repayment duration, financial term, and associated penalty.</p>
      *
-     * @param rating             The player's performance rating as an integer. Defaults to higher
-     *                           loan penalties and stricter terms for lower ratings.
-     * @param simulateGrayMonday A {@code boolean} flag that indicates whether the Gray Monday
-     *                           event is active, which impacts loan terms significantly.
-     * @param date               The current in-game date as a {@link LocalDate} object, used
-     *                           to determine if Gray Monday conditions apply.
+     * @param rating             The player's performance rating as an integer. Defaults to higher loan penalties and
+     *                           stricter terms for lower ratings.
+     * @param simulateGrayMonday A {@code boolean} flag that indicates whether the Gray Monday event is active, which
+     *                           impacts loan terms significantly.
+     * @param campaign           The current campaign object, used for LocalDate to determine if Gray Monday conditions
+     *                           apply and for fetching faction.
      *
-     * @return A {@link Loan} object representing the player's base loan terms based on
-     *         their rating and event conditions.
+     * @return A {@link Loan} object representing the player's base loan terms based on their rating and event
+     *       conditions.
      */
-    public static Loan getBaseLoan(final int rating, boolean simulateGrayMonday, final LocalDate date) {
+    public static Loan getBaseLoan(final int rating, boolean simulateGrayMonday, final Campaign campaign) {
         // we are going to treat the score from StellarOps the same as dragoons score
-        // TODO: pirates and government forces
 
+        LocalDate date = campaign.getLocalDate();
         if (isGrayMonday(date, simulateGrayMonday)) {
             // This simulates the player taking out a predatory loan
             return new Loan(10000000, 60, 1, FinancialTerm.MONTHLY, 100, date);
         }
-
-        if (rating <= 0) {
+        if (campaign.getFaction().isGovernment()) {
+            return new Loan(10000000, 5, 1, FinancialTerm.MONTHLY, 0, date);
+        } else if (campaign.getFaction().isPirate()) {
```

Comment:
If you need the faction from `Campaign#getFaction()` you pass the faction as argument, not campaign